shared_model/backend/protobuf/impl/proto_query_response_factory.cpp

```diff
@@ -196,6 +196,32 @@ shared_model::proto::ProtoQueryResponseFactory::createTransactionsResponse(
       query_hash);
 }
 
+std::unique_ptr<shared_model::interface::QueryResponse>
+shared_model::proto::ProtoQueryResponseFactory::createTransactionsPageResponse(
+    std::vector<std::unique_ptr<shared_model::interface::Transaction>>
+        transactions,
+    const crypto::Hash &next_tx_hash,
+    interface::types::TransactionsNumberType all_transactions_size,
```

Comment:
Don't like name `all_transactions_size`. What exactly does this field in proto mean? Number of transactions, which are to be retrieved via the query? Maybe, it's better then to use `requested_transactions_amount` or something close.

Reply:
All transactions size is the total number of transactions in all pages. Any better idea for the naming?